app/controllers/admin/orders_controller.rb

```diff
@@ -1,4 +1,35 @@
 class Admin::OrdersController < Admin::BaseController
+
   def index
+    # user = User.find_by(username: params[:session][:username])
+    # if user.current_admin
+    # @orders = Order.all
+    session[:orders] = nil
+    @orders = get_orders
+    # byebug
+      # byebug
+    # else
+    #   render:
+    # end
+    @status = Order.group(:status).count
+  end
+
+  def sort_orders
```

Comment:
Here is a put method, the way I have it set up right now when you use the dropdown on the admin order index you can get the status that we are trying to sort by.